scripts/build.py

```diff
@@ -0,0 +1,19 @@
+import shutil
+from pathlib import Path
+
+import click
+from ape import project
+
+PLUGIN_MANIFEST_FOLDER = project.path / "ape_safe" / "manifests"
+
+
+@click.command()
+def cli():
```

Comment:
nit: adding a little doc str here would be the script's CLI help text look nicer